### PR TITLE
Adjust indentation

### LINKED ISSUE #1751: Consistent indendation in Nemo
I think it is somewhat annoying that Nemo does not use a consistent indentation throughout the project. Could we agree on one specific indentation, and push that throughout the project?

### PR DESCRIPTION
**Pull request by joschmitt**
I found the magical bash/vim one-liner that adjusts the indentation to two spaces in all files.

Fixes #1751.
Closes #1752 (that is, I leave other formatting rules to somebody else)

### CODE REVIEW

**Comment by thofma** [comment]
Good enough for me. The low coverage is frightening, but let's ignore this for now.

File: src/flint/fmpz.jl [hunk 1]
```diff
@@ -164,20 +164,20 @@ is_positive(n::ZZRingElem) = sign(Int, n) > 0
...
+                                         (Ref{ZZRingElem},), a)
 
 @doc raw"""
     fits(::Type{UInt}, a::ZZRingElem)
 
 Return `true` if $a$ fits into a `UInt`, otherwise return `false`.
 """
 fits(::Type{UInt}, a::ZZRingElem) = a < 0 ? false :
-              ccall((:fmpz_abs_fits_ui, libflint), Bool, (Ref{ZZRingElem},), a)
+ccall((:fmpz_abs_fits_ui, libflint), Bool, (Ref{ZZRingElem},), a)
```
[the remark this answers is not in the document]
Comment (joschmitt) [reply src/flint/fmpz.jl]: I refactored that functions. If it doesn't fit in a line, one maybe shouldn't write it in one line :)

**Comment by lgoettgens** [comment]
@joschmitt do you want this merged now, or only after #1746 and #1754 are resolved?

**Comment by joschmitt** [comment]
> @joschmitt do you want this merged now, or only after #1746 and #1754 are resolved?

Let's merge this. It's never gonna be painless.

File: src/flint/fq_default_extended.jl [hunk 2]
```diff
@@ -828,7 +828,7 @@ such that `parent(a)(lift(R, a)) == a` is `true`.
 """
 function lift(R::FqPolyRing, a::FqFieldElem)
   base_ring(R) !== base_field(parent(a)) &&
-      error("Polynomial ring has wrong coefficient ring")
+  error("Polynomial ring has wrong coefficient ring")
```
Comment (thofma) [inline src/flint/fq_default_extended.jl:831]: this pr here has the same problem

**Pull request merged**